pkg/log/messages/error_messages.go

```diff
@@ -58,7 +58,7 @@ const CSPFK038E string = "CSPFK038E Retransmission backoff exhausted"
 const CSPFK039E string = "CSPFK039E Secrets Provider for Kubernetes failed to update secrets in %s mode. Reason: %s"
 
 // Annotations
-const CSPFK040E string = "CSPFK040E Failed to parse annotations file"
+//const CSPFK040E string = "CSPFK040E Failed to parse annotations file"
```

Comment:
Is there a reason to leave this line commented out instead of deleting it?

Reply:
No, I will delete it.